api/v1/report.proto

```diff
@@ -114,33 +114,32 @@ message LogEntry {
     EMERGENCY = 800;
   }
 
+  // The name of the LogEntryDescriptor providing the schema for this log entry.
+  string descriptor_name = 1;
+
   // Destination log for this entry
```

Comment:
not relevant to this review, but I wonder if we need to clarify terminology here a bit. something like:

"Destination collection for this log entry".